python/vision_explanation_methods/evaluation/pointing_game.py

```diff
@@ -41,7 +42,7 @@ def pointing_game(self,
                       imagelocation: str,
                       index: int,
                       threshold: float = .8,
-                      num_masks: int = 100):
+                      num_masks: int = 300):
```

Comment:
nit: move numbers to constants, perhaps at top of file, eg:
NUM_MASKS = 300
and reuse here as:
num_masks: int = NUM_MASKS
and elsewhere

Reply:
then in other code that calls this method we can specify the constants where needed